link/src/main/java/com/stripe/android/link/ui/wallet/WalletScreen.kt

```diff
@@ -174,13 +165,8 @@ internal fun WalletBody(
 
 @Composable
 internal fun WalletBody(
-    paymentDetailsList: List<ConsumerPaymentDetails.PaymentDetails>,
-    supportedTypes: Set<String>,
-    selectedItem: ConsumerPaymentDetails.PaymentDetails?,
-    isExpanded: Boolean,
+    uiState: WalletUiState,
     primaryButtonLabel: String,
```

Comment:
Ideally, we’d also be able to move `primaryButtonLabel` to the UI state. However, that’s a little more effort because it’s relying on `Context` and because the `completePaymentButtonLabel` method is used in other places. Therefore, I’m keeping this pull request focused on just the wallet screen and will keep this in mind for a future improvement.